src/compositemark/common.ts

```diff
@@ -123,11 +123,16 @@ export function getCompositeMarkTooltip(
     }
   );
 
+  const tooltipFieldDefs = fieldDefs(encodingWithoutContinuousAxis).map(fd =>
+    // remove guides as they don't work for tooltips
+    omit(fd, ['legend', 'axis', 'header'] as any[])
```

Comment:
What about scale?  

I also look at `tooltip.ts`, it seems like our tooltip logic reads `title` for its key.  So simply drop guides here is also incorrect.

Reply:
Oh, we read the title off the guide? We should move the title to the encoding property then.

Reply:
Yeah, I'd say look at tooltip.ts carefully and make sure we keep all important bits.